### DISCUSSION TITLE
issue: Re-Validation is not working with the setValue function

### DISCUSSION THREAD

**Discussion by serhat-m** [react-hook-form/react-hook-form#9580] [Q&A] [answered]
### Version Number

7.40.0

### Codesandbox/Expo snack

https://codesandbox.io/s/react-hook-form-validation-bug-gnekhi

### Steps to reproduce

1. Submit the form without any content to create an error
2. If you write directly into the input field, then the re-validation works
3. If you update the value with the setValue function, then re-ralidation does not work

![React-Hook-Form-Problem](https://user-images.githubusercontent.com/51929566/207299732-5ab4a670-3e11-41ef-8f0b-b08976dc3d9d.gif)

### Expected behaviour

The setValue function should re-validate as if you were manipulating the input field yourself with the keyboard

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by bluebill1049** [comment, marked as answer]
by default `setValue` doesn't validate to save computation: https://react-hook-form.com/api/useform/setvalue

`setValue('name', 'value', { shouldValidate: true })`

**Reply by serhat-m** [reply]
Thank you! Unfortunately I have overlooked that.
